core/src/main/kotlin/org/evomaster/core/output/clustering/metrics/DistanceMetricLastLine.kt

```diff
@@ -0,0 +1,23 @@
+package org.evomaster.core.output.clustering.metrics
+
+import org.evomaster.core.problem.rest.RestCallResult
+
+/**
+ * Distance metric implementation for clustering Rest Call Results.
+ *
+ * The distance measurement is based on the Levenshtein distance, normalized by length.
+ * Longer term, a means of weighting the clustering (to cluster first based on the module of the last executed line,
+ * then class, then line, maybe?) can be attempted.
+ *
+ * The idea is to use the path to the last executed line in the SUT for clustering.
+ *
+ */
+
+class DistanceMetricLastLine : DistanceMetric<RestCallResult>() {
+    override fun calculateDistance(val1: RestCallResult, val2: RestCallResult): Double {
```

Comment:
considering Kotlin uses `val` as keyword, calling variables `val1` is bit confusing.
Could use `a` and `b`, or `first` and `second`